store/event-store/src/main/java/org/calrissian/accumulorecipes/eventstore/impl/AccumuloEventStore.java

```diff
@@ -301,4 +301,45 @@ public void flush() throws Exception {
     public CloseableIterable<String> getTypes(String prefix, Auths auths) {
         return helper.getKeyValueIndex().getTypes(prefix, auths);
     }
+
+    public static class Builder {
```

Comment:
Not really that formal, just things get messy quickly if there are a bunch of one-offs. Considering how close the event and entity stores are API wise it makes sense to keep those in sync, but not that big a deal to include builders for all the stores.